src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +103,27 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (someDate.isAfter(LocalDate.now())) {
+            return someDate + " is after " + LocalDate.now();
+        } else if (someDate.isBefore(LocalDate.now())) {
+            return someDate + " is before " + LocalDate.now();
+        } else {
+            return someDate + " is today";
+        }
     }
 
     /**
      * Даны две временные зоны.
      * Верните Optional часовой разницы между двумя временными зонами.
+     *
      * @return Optional positive Integer
      */
     public Optional<Integer> diffBetweenZones(String firstZone, String secondZone) {
-        return Optional.empty();
+        int now = (int) System.currentTimeMillis();
```

Comment:
let's simplify this method using `ZoneId.of(firstZone)` and `ZoneId.of(secondZone)`